resources/windows/wix/ui/WixUI_PuppetInstallDir.wxs.erb

```diff
@@ -93,15 +91,10 @@ Patch dialog sequence:
 
       <Publish Dialog="PuppetWelcomeDlg" Control="Next" Event="NewDialog" Value="PuppetLicenseAgreementDlg"><![CDATA[NOT VersionNT64 AND NOT Installed]]></Publish>
       <Publish Dialog="PuppetWelcomeDlg" Control="Next" Event="NewDialog" Value="VerifyReadyDlg"><![CDATA[NOT VersionNT64 AND Installed AND PATCH]]></Publish>
-      <Publish Dialog="PuppetWelcomeDlg" Control="Next" Event="NewDialog" Value="Puppet32bitWarningDlg"><![CDATA[VersionNT64]]></Publish>
```

Comment:
These removals all look good to me just looking at the code - an installer will tell us for sure.